Controllers/ControllerFacade.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace covid19_api.Controllers
+{
+    /// <summary>
+    /// implementation of generic facade controller
+    /// </summary>
+    /// <seealso cref="Microsoft.AspNetCore.Mvc.ControllerBase" />
+    [ApiController]
+    public class ControllerFacade : ControllerBase
```

Comment:
Can you explain why do you think we need this Facade?
What are the benefits?

Reply:
sure, its a faćade patttern design , which in few works in meant to be inherith in the controllers layer to stop writting repetive code, for example the [apicontroller] attribute , others more important like if we were used to use CQRS or mediator , you can inject this type of props for every controller.